### PR TITLE
Simple class to use for colors

### PR DESCRIPTION
**Pull request by Joergenson**
Just add a name and an RGB value. So far I have added the color styles from Figma

### CODE REVIEW

File: components/colors.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default {
+    white: 'rgba(255, 255, 255, 1)',
+    black: 'rgba(0, 0, 0, 1)',
+    lightGray: 'rgba(234, 234, 234, 1)',
+    darkGray: 'rgba(76, 80, 85, 1)',
+    gray: 'rgba(139, 144, 147, 1)',
+    gradientColor1: 'rgba(223, 122, 153, 1)',
```
Comment (Kechr) [inline components/colors.js:7]: Maybe a more describing name would be suitable here? I think this is pinkish, while the other one is kinda yellow.

**Review by Kechr — COMMENTED** [review]
Looks nice!

File: components/colors.js (hunk 1 above)
Comment (Joergenson) [reply components/colors.js:7]: Yeah, just named them that because it's the only place it's used.

Comment (Joergenson) [reply components/colors.js]: Should be updated now 😎

**Review by Kechr — APPROVED** [review]
Sweet naming 😎

**Review by kepet19 — APPROVED** [review]

**Pull request merged**